packages/fxa-payments-server/src/routes/Product/SubscriptionRedirect/index.tsx

```diff
@@ -24,26 +24,36 @@ export const SubscriptionRedirect = ({ plan }: SubscriptionRedirectProps) => {
     downloadURL || productRedirectURLs[product_id] || defaultProductRedirectURL;
 
   useEffect(() => {
-    navigateToUrl(redirectUrl);
-  }, [navigateToUrl, redirectUrl]);
+    window.addEventListener('message', handleIframeTask);
+  });
+
+  const handleIframeTask = (e: any) => {
+    if (e.data === 'submitted survey') {
+      console.log('survey: ' + e.data);
```

Comment:
Should snip out this `console.log` before merge

Reply:
done.